packages/fluentui/react-builder/src/utils/codeToTree.ts

```diff
@@ -47,14 +48,26 @@ export const codeToTree: (code: string) => JSONTreeElement = code => {
       }
     }
 
+    // eslint-disable-next-line
+    const isElement = 'A' <= name[0] && name[0] <= 'Z';
```

Comment:
```suggestion
    const isElement = /^[A-Z]./.test(name);
```

Reply:
Is this enough to say if it's `Element`?

Reply:
That is a good question. It is not from my head, that is how we were deciding whether to add the `$$typeof` prop previously, I have just put the statement into a variable. The problem with this is that if user writes something like `<Div>` into the code, the UI builder will consider it as an element. The only solution how to fix it that I can think of is to check whether the `name` is in the array of possible objects to drag and drop. But that would cause problems for example for Icons.

Reply:
Would it work if we check with `React.isValidElement`?

Reply:
I don't think so, we only have access to the name, for example: 'div', 'Button'. To my understanding `React.isValidElement` is working with objects?

Reply:
Yeah, only works if we get the instance itself, so let's leave it as it's for now :)